pkg/util/validation/limits.go

```diff
@@ -899,6 +901,11 @@ func (o *Overrides) RulerSyncRulesOnChangesEnabled(userID string) bool {
 	return o.getOverridesForUser(userID).RulerSyncRulesOnChangesEnabled
 }
 
+// RulerMaxConcurrentRuleEvaluationsPerTenant returns the maximum number of independent rules that can run concurrently for a given user.
+func (o *Overrides) RulerMaxConcurrentRuleEvaluationsPerTenant(userID string) int64 {
```

Comment:
[nit] You can remove "PerTenant" from function name too.

Reply:
We agreed offline to leave it in because otherwise we get conflicting flags as the global and per tenant one would have the same name.